src/main/java/hudson/plugins/robot/view/RobotListViewColumn.java

```diff
@@ -41,12 +41,26 @@ public long getTotal(Item job){
 		return 0;
 	}
 
+	public long getSkipped(Item job) {
+		RobotResult lastRobotResult = getLastRobotResult(job);
+		if (lastRobotResult != null) {
+			return lastRobotResult.getOverallSkipped();
+		}
+		return 0;
+	}
+
 	public double getPassPercent(Item job) {
 		RobotResult lastRobotResult = getLastRobotResult(job);
 		if (lastRobotResult==null) return 100;
 		return lastRobotResult.getPassPercentage();
 	}
 
+	public double getSkipPercent(Item job) {
+		RobotResult lastRobotResult = getLastRobotResult(job);
+		if (lastRobotResult == null) return 0;
```

Comment:
Please use braces :)

Reply:
How about a few lines up, where this is not used (not changed in this PR).

Reply:
Please use it there as well :)

Reply:
These are outdated. All these are changed already.

Reply:
Right, got confused, my bad. You resolved it the other way I thought